netmap/netmap_contract.go

```diff
@@ -312,10 +316,13 @@ func UpdateState(state int, publicKey interop.PublicKey) {
 		common.CheckAlphabetWitness(common.AlphabetAddress())
 	}
 
-	switch nodeState(state) {
+	switch NodeState(state) {
 	case OfflineState:
 		removeFromNetmap(ctx, publicKey)
 		runtime.Log("remove storage node from the network map")
+	case MaintenanceState:
```

Comment:
previously we use that method only for removing from netmap since Online == `AddPeer`

now we have a new scenario: `Maintenance` => `Online`. won't it fail with `panic("unsupported state")`? think doesn't matter whether I'm right or not, we could also add that case in our tests since it gonna be a regular case